module/zfs/arc.c

```diff
@@ -463,6 +463,11 @@ int zfs_arc_meta_strategy = ARC_STRATEGY_META_BALANCED;
 int zfs_arc_meta_adjust_restarts = 4096;
 int zfs_arc_lotsfree_percent = 10;
 
+/*
+ * Number of arc_prune threads
+ */
+int zfs_arc_prune_task_threads = 1;
```

Comment:
This can (should) be `static`.

Reply:
Sure. Thanks. Though there I see plenty of other variables that are not static.

Reply:
This is because I spent like seven-plus hours yesterday/today-morning making them so (and, where possible, const): #12899.